shoes-package/lib/warbler/traits/shoes.rb

```diff
@@ -29,15 +29,37 @@ def self.requires?(trait)
         [Traits::Jar, Traits::NoGemspec].include? trait
       end
 
+      INIT_CONTENTS = <<~EOS
+        require 'shoes'
+        require 'shoes/swt'
+
+        Shoes::Swt.initialize_backend
+      EOS
+
       def after_configure
-        config.init_contents << StringIO.new("require 'shoes'\nrequire 'shoes/swt'\nShoes::Swt.initialize_backend\n")
+        config.init_contents << StringIO.new(INIT_CONTENTS)
       end
 
       def update_archive(jar)
         super
         add_main_rb(jar, apply_pathmaps(config, default_executable, :application))
```

Comment:
I... don't understand. We've been calling this method all along and just now we implement it? I'm... I don't know. Magic? Inheritance? What?